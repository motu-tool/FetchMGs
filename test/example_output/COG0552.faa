version https://git-lfs.github.com/spec/v1
oid sha256:57f6cb67cf0c057cca8e5967b1862b749c047af31251ad78ee537b85e9577ac8
size 5788
