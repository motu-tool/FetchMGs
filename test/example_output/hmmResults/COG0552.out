# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.3.2 (Nov 2020); http://hmmer.org/
# Copyright (C) 2020 Howard Hughes Medical Institute.
# Freely distributed under the BSD open source license.
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0552.hmm
# target sequence database:        example_input/example_data_genomes.faa
# output directed to file:         example_output/hmmResults/COG0552.out
# per-dom hits tabular output:     example_output/hmmResults/COG0552.dom
# show alignments in output:       no
# max ASCII text line length:      unlimited
# sequence reporting threshold:    score >= 220
# number of worker threads:        1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       COG0552  [M=334]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence           Description
    ------- ------ -----    ------- ------ -----   ---- --  --------           -----------
   2.2e-136  454.6   0.0   2.3e-136  454.5   0.0    1.0  1  1089456.NCGM2_5826  protein_id="BAK92637.1" gene="ftsY" product="signal recognition particle receptor" GI="348037277" note="similar to GI:116054103; similar to PA14_04900 of P. aeruginosa UCBPP-PA14" project_id="73453"
   6.8e-119  397.1   6.4   8.9e-119  396.7   6.4    1.0  1  1037409.BJ6T_04420  protein_id="BAL05740.1" GI="354953061" note="probable cell division protein" project_id="67463"
   3.8e-118  394.6   1.7   4.2e-118  394.5   1.7    1.1  1  1148.SYNGTS_1420    protein_id="BAK50168.1" gene="ftsY" product="cell division protein FtsY" GI="339273681" note="ORF_ID:slr2102" project_id="67081"
   2.3e-105  352.7   5.1   2.9e-105  352.3   5.1    1.1  1  944547.ABLL_1126    protein_id="BAK73001.1" product="signal recognition particle-docking GTPase FtsY" GI="345471551" project_id="61721"
   2.7e-105  352.4   4.4   3.6e-105  352.0   4.4    1.1  1  944546.ABED_0841    protein_id="BAK70558.1" product="signal recognition particle-docking GTPase FtsY" GI="345469107" project_id="61719"
    1.1e-85  287.9   6.0    1.1e-85  287.9   6.0    1.3  1  1089456.NCGM2_4879  protein_id="BAK91702.1" gene="ffh" product="signal recognition particle protein" GI="348036342" note="similar to GI:116051744; similar to PA14_15960 of P. aeruginosa UCBPP-PA14" project_id="73453"
    3.6e-83  279.7   7.6    4.2e-83  279.4   7.6    1.1  1  1148.SYNGTS_1731    protein_id="BAK50479.1" gene="ffh" product="signal recognition particle protein" GI="339273992" note="ORF_ID:slr1531" project_id="67081"
    3.6e-80  269.8   6.0      5e-80  269.3   6.0    1.2  1  944547.ABLL_2406    protein_id="BAK74281.1" product="signal recognition particle protein" GI="345472831" project_id="61721"
    8.1e-78  262.1   9.7    8.6e-78  262.0   8.6    1.3  1  944546.ABED_1783    protein_id="BAK71500.1" product="signal recognition particle protein" GI="345470049" project_id="61719"
    2.8e-73  247.2   8.3    3.1e-73  247.0   8.3    1.2  1  1037409.BJ6T_04500  protein_id="BAL05748.1" product="signal recognition particle protein" GI="354953069" project_id="67463"


Domain annotation for each sequence:
>> 1089456.NCGM2_5826  protein_id="BAK92637.1" gene="ftsY" product="signal recognition particle receptor" GI="348037277" note="similar to GI:116054103; similar to PA14_04900 of P. aeruginosa UCBPP-PA14" project_id="73453"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  454.5   0.0  9.8e-140  2.3e-136      20     325 ..     149     453 ..      22     456 .] 0.92

>> 1037409.BJ6T_04420  protein_id="BAL05740.1" GI="354953061" note="probable cell division protein" project_id="67463"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  396.7   6.4  3.8e-122  8.9e-119      20     322 ..      12     311 ..       5     314 .. 0.97

>> 1148.SYNGTS_1420  protein_id="BAK50168.1" gene="ftsY" product="cell division protein FtsY" GI="339273681" note="ORF_ID:slr2102" project_id="67081"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  394.5   1.7  1.8e-121  4.2e-118      19     323 ..     185     503 ..      67     504 .] 0.89

>> 944547.ABLL_1126  protein_id="BAK73001.1" product="signal recognition particle-docking GTPase FtsY" GI="345471551" project_id="61721"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  352.3   5.1  1.2e-108  2.9e-105      20     324 ..      53     343 ..       3     350 .] 0.88

>> 944546.ABED_0841  protein_id="BAK70558.1" product="signal recognition particle-docking GTPase FtsY" GI="345469107" project_id="61719"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  352.0   4.4  1.5e-108  3.6e-105      26     324 ..      21     304 ..       1     308 [. 0.86

>> 1089456.NCGM2_4879  protein_id="BAK91702.1" gene="ffh" product="signal recognition particle protein" GI="348036342" note="similar to GI:116051744; similar to PA14_15960 of P. aeruginosa UCBPP-PA14" project_id="73453"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  287.9   6.0   4.8e-89   1.1e-85      22     324 ..       2     297 ..       1     315 [. 0.92

>> 1148.SYNGTS_1731  protein_id="BAK50479.1" gene="ffh" product="signal recognition particle protein" GI="339273992" note="ORF_ID:slr1531" project_id="67081"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  279.4   7.6   1.8e-86   4.2e-83      22     325 ..       2     297 ..       1     402 [. 0.89

>> 944547.ABLL_2406  protein_id="BAK74281.1" product="signal recognition particle protein" GI="345472831" project_id="61721"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  269.3   6.0   2.2e-83     5e-80      38     323 ..      11     289 ..       2     394 .. 0.89

>> 944546.ABED_1783  protein_id="BAK71500.1" product="signal recognition particle protein" GI="345470049" project_id="61719"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  262.0   8.6   3.7e-81   8.6e-78      39     323 ..      12     289 ..       2     318 .. 0.91

>> 1037409.BJ6T_04500  protein_id="BAL05748.1" product="signal recognition particle protein" GI="354953069" project_id="67463"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  247.0   8.3   1.4e-76   3.1e-73      24     325 ..       4     298 ..       1     422 [. 0.87



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (334 nodes)
Target sequences:                      23272  (7230323 residues searched)
Passed MSV filter:                      1642  (0.0705569); expected 465.4 (0.02)
Passed bias filter:                     1316  (0.0565486); expected 465.4 (0.02)
Passed Vit filter:                       398  (0.0171021); expected 23.3 (0.001)
Passed Fwd filter:                       111  (0.00476968); expected 0.2 (1e-05)
Initial search space (Z):              23272  [actual number of targets]
Domain search space  (domZ):              10  [number of targets reported over threshold]
# CPU time: 0.58u 0.01s 00:00:00.59 Elapsed: 00:00:00.47
# Mc/sec: 5056.02
//
[ok]
